modules/oit/include/modules/oit/datastructures/rasterization.h

```diff
@@ -80,6 +84,14 @@ class IVW_MODULE_OIT_API Rasterization {
      */
     std::optional<mat4> boundingBox() const;
 
+    struct RaycastingState {

```

Comment:
should this not be in the Rasterizer class instead? 

why do we need to have it here?

Reply:
Moved to `rasterizer.h` as standalone struct.